src/screens/CreatePost/index.js

```diff
@@ -449,11 +450,13 @@ const CreatePost = () => {
   };
 
   const postV2 = async () => {
+    setLoadingPost(true);
     if (!isEmptyMessageAllowed()) {
       showMessage({
         message: StringConstant.createPostFailedNoMessage,
         type: 'danger'
       });
+      setLoadingPost(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `setLoadingPost(true)` before checking if the message is empty and then setting it back to false if the message is indeed empty is a good practice. It ensures that the loading state is properly managed even when the message is empty.
```diff
+    setLoadingPost(true);
+     if (!isEmptyMessageAllowed()) {
+       showMessage({
+         message: StringConstant.createPostFailedNoMessage,
+         type: 'danger'
+       });
+      setLoadingPost(false);
+      return true;
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->